### ISSUE TITLE
laravel 9 api routes error

### ISSUE THREAD

**Issue by dangermember** [CrestApps/laravel-code-generator#186]
the command i used
php artisan create:api-scaffold User --with-api-resource

the use at the top of api.php
use App\Api\Http\Controllers\Api\Api\UsersController;
supposed to be
use App\Http\Controllers\Api\UsersController;

data from my composer.json
        "php": "^8.0.2",
        "laravel/framework": "^9.19",
        "crestapps/laravel-code-generator": "^2.4",

**Comment by MikeAlhayek** [comment]
Are you using the default setting or have you customized the settings?

**Comment by dangermember** [comment]
My config
    'files_upload_path' => 'uploads',
    'datetime_out_format' => 'Y-m-d H:i:s',
    'template'=>'default'

**Comment by MikeAlhayek** [comment]
I'll have to look more at the code. But can you try adding this to your config?
```
'api_controllers_path' => 'Http/Controllers',
```


Does that fix it?

**Comment by dangermember** [comment]
First i would like to thank you for your fast response.

when using config:
`'api_controllers_path' => 'Http/Controllers',`
the output is:
```
use App\Http\Controllers\UsersController;

Route::group([
    'prefix' => 'users',
], function () {
    Route::get('/', [UsersController::class, 'index'])
         ->name('api.users.user.index');
    Route::get('/show/{user}',[UsersController::class, 'show'])
         ->name('api.users.user.show');
    Route::post('/', [UsersController::class, 'store'])
         ->name('api.users.user.store');    
    Route::put('user/{user}', [UsersController::class, 'update'])
         ->name('api.users.user.update');
    Route::delete('/user/{user}',[UsersController::class, 'destroy'])
         ->name('api.users.user.destroy');
});

```
when using config:
`'api_controllers_path' => 'Http/Controllers/API',`
the output is:
```
use App\API\Http\Controllers\API\API\UsersController;
Route::group([
    'prefix' => 'users',
], function () {
    Route::get('/', [API\UsersController::class, 'index'])
         ->name('api.users.user.index');
    Route::get('/show/{user}',[API\UsersController::class, 'show'])
         ->name('api.users.user.show');
    Route::post('/', [API\UsersController::class, 'store'])
         ->name('api.users.user.store');    
    Route::put('user/{user}', [API\UsersController::class, 'update'])
         ->name('api.users.user.update');
    Route::delete('/user/{user}',[API\UsersController::class, 'destroy'])
         ->name('api.users.user.destroy');
});
```

while the expected output is:
```
use App\Http\Controllers\API\UsersController;
Route::group([
    'prefix' => 'users',
], function () {
    Route::get('/', [UsersController::class, 'index'])
         ->name('api.users.user.index');
    Route::get('/show/{user}',[API\UsersController::class, 'show'])
         ->name('api.users.user.show');
    Route::post('/', [UsersController::class, 'store'])
         ->name('api.users.user.store');    
    Route::put('user/{user}', [UsersController::class, 'update'])
         ->name('api.users.user.update');
    Route::delete('/user/{user}',[UsersController::class, 'destroy'])
         ->name('api.users.user.destroy');
});
```

**Comment by MikeAlhayek** [comment]
Not sure. I'll test it and respond with my findings.

**Comment by MikeAlhayek** [comment]
@dangermember this fix is part of v2.4.9 now.

**Comment by dangermember** [comment]
Thank you very much.
